javascript3/week2/exercise.html

```diff
@@ -1,5 +1,5 @@
 <audio
-  src="https://raw.githubusercontent.com/mdn/webaudio-examples/master/audio-basics/outfoxing.mp3"
+  src="https://raw.githubusercontent.com/mdn/webaudio-examples/main/audio-basics/outfoxing.mp3"
```

Comment:
@martamatos can you double check this.

 I think this external like doesn't work with main raw.githubusercontent.com/mdn/webaudio-examples/main/audio-basics/outfoxing.mp3.  
 
 Can we keep raw.githubusercontent.com/mdn/webaudio-examples/master/audio-basics/outfoxing.mp3 ?